lib/stripe/api_operations/save.rb

```diff
@@ -66,7 +66,7 @@ def save(params = {}, opts = {})
         # generated a uri for this object with an identifier baked in
         values.delete(:id)
 
-        resp, opts = execute_resource_request(:post, save_url, values, opts)
+        resp, opts = execute_resource_request(:post, save_url, values, opts, ["save"])
```

Comment:
I'm fine with hard coding this now, but when we want to report more features, we'll probably need to change how we keep track of a running `usage` list.

Reply:
Agreed! The semantics right now is for `execute_resource_request` to expect its caller to pass in all the usages it knows about, and then `execute_resource_request` will be responsible for appending any additional usage that happens at the infrastructure level (right now that's none), so I think the interface is future-proof but the implementation might need to change.